components/formats-gpl/src/loci/formats/in/SlideBook6Reader.java

```diff
@@ -0,0 +1,388 @@
+import loci.formats.FormatException;
+import loci.formats.FormatReader;
+import loci.formats.FormatTools;
+import loci.formats.MetadataTools;
+import loci.formats.meta.MetadataStore;
+
+import ome.xml.model.primitives.PositiveFloat;
+import ome.units.quantity.Length;
+import ome.units.quantity.Time;
+import ome.units.UNITS;
+
+import loci.formats.MissingLibraryException;
+
+/**
+ * SlideBook6Reader is a file format reader for 3i SlideBook SLD files that uses
+ * the SlideBook SBReadFile SDK via the (Windows only) dlls: 
+ *   SBReadFile.dll and SlideBook6Reader.dll
+ *
+ * @author Richard Myers, richard at intelligent-imaging.com
+ *
+ */
+public class SlideBook6Reader  extends FormatReader {
+
+	// -- Constants --
+
+	public static final int SLD_MAGIC_BYTES_1_0 = 0x006c;
+	public static final int SLD_MAGIC_BYTES_1_1 = 0x0100;
+	public static final int SLD_MAGIC_BYTES_1_2 = 0x0200;
+	public static final int SLD_MAGIC_BYTES_2_0 = 0x01f5;
+	public static final int SLD_MAGIC_BYTES_2_1 = 0x0102;
+
+	public static final long SLD_MAGIC_BYTES_3 = 0xf6010101L;
+
+	private static final String URL_3I_SLD =
+			"http://www.openmicroscopy.org/site/support/bio-formats/formats/3i-slidebook6-sld.html";
+	private static final String NO_3I_MSG = "3i SlideBook SBReadFile library not found. " +
+			"Please see " + URL_3I_SLD + " for details.";
+	private static final String GENERAL_3I_MSG = "3i SlideBook SBReadFile library problem. " +
+			"Please see " + URL_3I_SLD + " for details.";
+
+	// -- Static initializers --
+
+	private static boolean libraryFound = false;
+
+	static {
+		String library_path = System.getProperty("java.library.path");
+
+		try {
+			// load JNI wrapper of SBReadFile.dll
+			System.loadLibrary("SlideBook6Reader");
+			libraryFound = true;
+		}
+		catch (UnsatisfiedLinkError e) {
+			LOGGER.warn(NO_3I_MSG, e);
+			libraryFound = false;
+		}
+		catch (SecurityException e) {
+			LOGGER.warn("Insufficient permission to load native library", e);
+			libraryFound = false;
+		}
+	}
+
+	// -- Constructor --
+
+	public SlideBook6Reader() {
+		super("SlideBook 6 SLD (native)", new String[] {"sld"});
+		domains = new String[] {FormatTools.LM_DOMAIN};
+		suffixSufficient = false;
+	}
+
+	// -- IFormatReader API methods --
+
+	/* @see loci.formats.IFormatReader#isThisType(RandomAccessInputStream) */
+	public boolean isThisType(RandomAccessInputStream stream) throws IOException {
+		final int blockLen = 8;
+		stream.seek(4);
+		boolean littleEndian = stream.readString(2).equals("II");
+		if (!FormatTools.validStream(stream, blockLen, littleEndian)) return false;
+		int magicBytes1 = stream.readShort();
+		int magicBytes2 = stream.readShort();
+
+		boolean isMatch = ((magicBytes2 & 0xff00) == SLD_MAGIC_BYTES_1_1 ||
+				(magicBytes2 & 0xff00) == SLD_MAGIC_BYTES_1_2) &&
+				(magicBytes1 == SLD_MAGIC_BYTES_1_0 ||
+				magicBytes1 == SLD_MAGIC_BYTES_2_0);
+
+		return isMatch;
+	}
+
+	/**
+	 * @see IFormatReader#isThisType(String, boolean)
```

Comment:
This `@see` reference is causing a warning in the Javadocs (see https://ci.openmicroscopy.org/job/BIOFORMATS-5.1-merge-build/861/warnings39Result/). You may to use

```
@see loci.formats.IFormatReader#isThisType(String, boolean)
```

as above